test/rules-engine/sssi.test.js

```diff
@@ -0,0 +1,60 @@
+describe('Rules engine rule test', () => {
+  let rulesEngine
+  const parcelsTestData = require('./test-data/parcels-sssi.json')
+  const testRules = require('./test-data/rules-sssi.json')
+  const successEvent = jest.fn()
+  const eventType = 'sssi'
+
+  beforeAll(async () => {
+    rulesEngine = require('../../server/rules-engine')
+  })
+
+  beforeEach(async () => {
+  })
+
+  test('sssi rule discards parcel with sssi', async () => {
+    rulesEngine.engine.on('success', async (event, almanac, ruleResult) => {
+      if (event.type === eventType) {
+        almanac.addRuntimeFact(rulesEngine.enums.ruleRejected, true)
+        successEvent()
+      }
+    })
+    const enabledRules = rulesEngine.enabledEligibilityRules(testRules)
+    const conditions = rulesEngine.conditionsFromRules(enabledRules)
+    const SSSIrule = rulesEngine.buildStandardRule(conditions, 'SSSI', eventType)
+    rulesEngine.engine.addRule(SSSIrule)
+    var actions = parcelsTestData.map(parcel => {
+      return rulesEngine.engine.run(parcel)
+    })
+    await Promise.all(actions)
+    expect(successEvent).toHaveBeenCalledTimes(1)
+  })
+
+  test('eligibility rules run gets parcels without sssi', async () => {
+    const acceptedParcels = []
+    const enabledRules = rulesEngine.enabledEligibilityRules(testRules)
+    const conditions = rulesEngine.conditionsFromRules(enabledRules)
+    rulesEngine.setupStandardRule(conditions, 'SSSI', eventType)
+    rulesEngine.setupAcceptedItemsRule(async function (event, almanac, ruleResult) {
+      acceptedParcels.push(await almanac.factValue('ref'))
+      successEvent()
+    })
+    var actions = parcelsTestData.map(parcel => {
+      return rulesEngine.engine.run(parcel)
+    })
+    await Promise.all(actions)
+
+    expect(acceptedParcels.includes('SD74445738')).toBe(true)
+    expect(acceptedParcels.includes('SD75492628')).toBe(true)
+    expect(acceptedParcels.includes('SD81437506')).toBe(true)
+    expect(acceptedParcels.includes('SD81525709')).toBe(true)
```

Comment:
This might be clearer if you declared `const sssiParcelRef = 'SD78379604'` at the start and asserted that it wasn't in the `acceptedParcels` array, because that's what the test is checking